sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -513,11 +513,11 @@ public partial class AzureTableDataFeedSource : Azure.AI.MetricsAdvisor.Models.D
     public partial class ChangeThresholdCondition
     {
         public ChangeThresholdCondition(double changePercentage, int shiftPoint, bool isWithinRange, Azure.AI.MetricsAdvisor.Models.AnomalyDetectorDirection anomalyDetectorDirection, Azure.AI.MetricsAdvisor.Models.SuppressCondition suppressCondition) { }
-        public Azure.AI.MetricsAdvisor.Models.AnomalyDetectorDirection AnomalyDetectorDirection { get { throw null; } }
-        public double ChangePercentage { get { throw null; } }
-        public bool IsWithinRange { get { throw null; } }
-        public int ShiftPoint { get { throw null; } }
-        public Azure.AI.MetricsAdvisor.Models.SuppressCondition SuppressCondition { get { throw null; } }
+        public Azure.AI.MetricsAdvisor.Models.AnomalyDetectorDirection AnomalyDetectorDirection { get { throw null; } set { } }
+        public double ChangePercentage { get { throw null; } set { } }
+        public bool IsWithinRange { get { throw null; } set { } }
+        public int ShiftPoint { get { throw null; } set { } }
+        public Azure.AI.MetricsAdvisor.Models.SuppressCondition SuppressCondition { get { throw null; } set { } }
```

Comment:
Added setters to improve the user experience during Update. Let's say user wants to update the `ChangePercentage` property to 50. Instead of doing:
```cs
ChangeThresholdCondition condition = client.Get(...);
var newCondition = new ChangeThresholdCondition(50, condition.ShiftPoint, condition.IsWithinRange, condition.AnomalyDetectorDirection, condition.SuppressCondition);
client.Update(newCondition);
```
They can do:
```cs
ChangeThresholdCondition condition = client.Get(...);
condition.ChangePercentage = 50;
client.Update(condition);
```

Reply:
love your example